app/views/pages/internal_server_error.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Sorry, there is a problem with the service</h1>
+    <p class="govuk-body">Try again later.</p>
+
+    <p class="govuk-body">
+      <a href="mailto:regionalservices.rg@education.gov.uk" class="govuk-link">Contact the Regional Services support team</a> if you have any questions.
```

Comment:
Given that we've used this email address in a few places, is it worth making it either an env variable or baking it into a constant somewhere? I'm not sure the language file is the right place, but it definitely feels more like something we should define and reuse than just type.